torchchat/utils/quantize.py

```diff
@@ -916,26 +970,11 @@ def quantized_model(self) -> nn.Module:
         torchao_experimental_quant_api
     )
     from torchao_experimental_quant_api import (
-        Int8DynActIntxWeightLinearQuantizer,
-        IntxWeightEmbeddingQuantizer,
         UIntxWeightOnlyLinearQuantizer,
     )
```

Comment:
```suggestion
        from torchao_experimental_quant_api import UIntxWeightOnlyLinearQuantizer
```

Drops down to single import, can lose the (...)